test/functional/world_locations_controller_test.rb

```diff
@@ -266,28 +265,15 @@ def assert_featured_editions(editions)
   view_test "should display translated page labels when requested in a different locale" do
     world_location = create(:world_location, translated_into: [:fr])
 
-    create(:published_worldwide_priority, world_locations: [world_location], translated_into: [:fr])
     create(:published_publication, world_locations: [world_location], translated_into: [:fr])
     create(:published_policy, world_locations: [world_location], translated_into: [:fr])
 
     get :show, id: world_location, locale: 'fr'
 
     assert_select ".type", "Localisation"
-    assert_select "#worldwide-priorities", /Priorités/
     assert_select "#publications .see-all a", /Voir toutes nos publications/
   end
 
-  test "should only display translated priorities when requested for a locale" do
-    world_location = create(:world_location, translated_into: [:fr])
-
-    translated_priority = create(:published_worldwide_priority, world_locations: [world_location], translated_into: [:fr])
-    untranslated_priority = create(:published_worldwide_priority, world_locations: [world_location])
-
-    get :show, id: world_location, locale: 'fr'
```

Comment:
If it's testing something we're removing, get rid of it, otherwise let's keep 'em.